pkg/services/accesscontrol/migrator/zanzana.go

```diff
@@ -253,3 +268,282 @@ func dashboardFolderCollector(store db.DB) TupleCollector {
 		return nil
 	}
 }
+
+// basicFixedRolesCollector migrates basic and fixed roles to OpenFGA tuples
+func basicFixedRolesCollector(store db.DB) TupleCollector {
```

Comment:
I wonder if our fixed roles could be part of the schema somehow. They are immutable.

Basic roles are a bit trickier and I agree that we should now have a basic role that spans multiple orgs. But whenever a new org is created we would need to set up them from a template or something

Reply:
That's good idea. I thought about making basic roles a part of schema, bit since they're modifiable, I did permissions migration. But seems fixed roles is a good candidate to put into schema.

Reply:
We can do that in another pr. I don't want to block this one. Maybe skip fixed roles for now and just add basic and custom roles. Then we can investigate fixed roles separately

Reply:
I think I would prefer to compose this as a separate PR.